index.js

```diff
@@ -52,6 +69,10 @@ addExceptionsToUnprotected(defaultConfig.features.contentBlocking.exceptions)
 
 if (!fs.existsSync(GENERATED_DIR)) {
     fs.mkdirSync(GENERATED_DIR)
+
+    // Create version directories
+    fs.mkdirSync(`${GENERATED_DIR}/v1`)
+    fs.mkdirSync(`${GENERATED_DIR}/v2`)
```

Comment:
Can we make the `existsSync` and `mkdirSync` a function and call it for all three directories? Locally this broke for me as I already had a generated/ but not the other dirs. Ideally users of the config shouldn't need to understand about the generation process.


Also worth noting this path is mentioned in the extensions CONTRIBUTING.md for local development.  (happy to add this to the task of integrating the update to v2)
Perhaps other projects are depending on the generated directory being the route?

Reply:
I'll get those into a function.

Regarding the change of local path yeah that's something we should probably delegate to v2 integration projects.